config/package/manifests/install.yaml

```diff
@@ -15,6 +15,12 @@ spec:
       labels:
         core.crossplane.io/name: "provider-aws"
     spec:
+      securityContext:
+        # TODO(hasheddan): this can be removed when there is no desire
+        # to support pre-v1.19 Kubernetes clusters
+        # https://github.com/kubernetes/enhancements/pull/1598
+        # https://github.com/aws/amazon-eks-pod-identity-webhook/issues/8
+        fsGroup: 65534
```

Comment:
is this fully compatible with the last official release of crossplane (v0.11)? i.e. if someone installs this latest provider-aws but hasn't updated crossplane to a newer master version, will it work OK?

Reply:
@jbw976 this will not break `master` of `provider-aws` with Crossplane `v0.11`/`v0.11.1`, but this will be stripped out meaning that the IAM Role for Service Accounts will not work until someone updates their Crossplane to `master`. So in short, this will not introduce any regressions, but will also not be a fix without updating Crossplane. (One could manually edit this as well I think to get the functionality with older Crossplane versions though).

Reply:
Yes that sounds perfect.  Totally fine with it not being "fixed" until their crossplane version is also updated. The important part is that they won't experience any crashes or complete breaks by using this new provider-aws update without also updating crossplane.